DESCRIPTION

```diff
@@ -3,15 +3,25 @@ Type: Package
 Title: ADaM in R Asset Library - Dev Tools
 Version: 0.0.1
 Authors@R: c(
-    person("Open", "Source", email = "admiralext@pharmaverse.com", role = c("aut", "cre")))
-Description: A toolbox for programming Clinical Data Standards Interchange Consortium
-    (CDISC) compliant Analysis Data Model (ADaM) datasets in R. ADaM datasets are a
-    mandatory part of any New Drug or Biologics License Application submitted to the
-    United States Food and Drug Administration (FDA). Analysis derivations are
-    implemented in accordance with the "Analysis Data Model Implementation Guide"
-    (CDISC Analysis Data Model Team, 2021, <https://www.cdisc.org/standards/foundational/adam/adamig-v1-3-release-package>).
+    person("Ben", "Straub", email = "ben.x.straub@gsk.com", role = c("aut", "cre")),
+    person("Stefan", "Bundfuss", role = "aut"),
+    person("Thomas", "Neitmann", role = "aut"),
+    person("Samia", "Kabi", role = "aut"),
+    person("Gordon", "Miller", role = "aut"),
+    person("Ross", "Farrugia", role = "aut"),
+    person("Annie", "Yang", role = "aut"),
+    person("Robin", "Koeger", role = "aut"),
+    person("Pooja", "Kumari", role = "aut"),
+    person("Alice", "Ehmann", role = "ctb"),
+    person("Vignesh", "Thanikachalam", role = "ctb"),
+    person("Ondrej", "Slama", role = "ctb"),
+    person("Shimeng", "Huang", role = "ctb"),
+    person("F. Hoffmann-La Roche AG", role = c("cph", "fnd")),
+    person("GlaxoSmithKline LLC", role = c("cph", "fnd"))
+    )
+Description: A toolbox for developers working on (pharmaverse admiral packages, <https://github.com/pharmaverse>). 
```

Comment:
Why the parentheses? Also this is too short. CRAN would reject it. Needs at least another sentence.

Reply:
I did it like this so becomes a hyperlink,  I just copied it from the admiral one for CDSIC.  I will add some more information

![image](https://user-images.githubusercontent.com/10111024/181831444-43324a73-0cd2-439b-ab29-43d3e1d48f3d.png)